app/lib/pdc_discovery/cli.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'thor'
+
+module PdcDiscovery
+  ##
+  # Command line interface for processing theses
+  class Cli < Thor
+    method_option :collection_handle, aliases: '-c', type: :string, desc: 'The handle identifier of the DataSpace collection to be indexed'
+
+    desc 'index', 'Index a DataSpace collection for discovery in the Princeton Data Commons'
+    def index
+      if all_required_options_present?
+        puts "Indexing for Princeton Data Commons"
+        output_options
+        # Indexer.index(options)
```

Comment:
Should this be commented? If this is delayed for another PR, I am sorry, I just wanted to be certain to verify this.